website/docs/guides/migration_guide_v3.html.markdown

```diff
@@ -0,0 +1,114 @@
+---
+layout: "newrelic"
+page_title: "New Relic Terraform Provider v3.x Migration Guide"
+sidebar_current: "docs-newrelic-provider-v3-migration-guide"
+description: |-
+  Use this guide to update the New Relic Terraform Provider from v2.x to v3.x
+---
+
+## Upgrade to v3.x of the New Relic Terraform Provider
+
+Version 3.x of the provider uses a new underlying API for Synthetics. This results in some changes that will need to be made to existing Synthetics resource to keep them compatible with the new API.
+
+### Migrating script Synthetics monitor resources
+
+The previous `newrelic_synthetics_monitor_resourece` has been split into two new resources: `newrelic_synthetics_monitor_resource` and `newrelic_synthetics_script_monitor_resource`. Previously a monitor script had to be defined separately from a Synthetics Monitor using the `newrelic_synthetics_monitor_script_resource`. In this new version a script is attached directly to a scripted monitor. See the below example for how to migrate to version 3:
```

Comment:
> The previous `newrelic_synthetics_monitor_resourece` 

resourece

Reply:
Should we use `v2` vs `v3` instead of previous and new version? It would make it clearer for me.